docs/application/dotnet/index.md

```diff
@@ -59,6 +59,8 @@ Tizen .NET consists of the following main components:
   [TizenFX API](api/overview.md) allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
 
   TizenFX supports C# APIs, which expose Tizen-specific features such as Tizen application framework, account management, location services, media services, and connectivity.
+  
+  TizenFX API provides NUI library for implementing GUI in both 2d and 3d modes. NUI is optimized for speed and low resource usage and may be used in wearables, tvs and mobile platforms.
```

Comment:
TizenFX API provides NUI library for implementing GUI in both 2D and 3D modes. NUI is optimized for speed and low resource usage, and can be used in wearables, TVs, and mobile platforms.